content/java/concepts/strings/terms/replace/replace.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.replace()'
+Description: 'Takes a string and returns a new string where a given char or CharSequence is replaced with another char or CharSequence.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.replace()` String method takes a string and replaces a given char or `CharSequence` with another char or `CharSequence`, returning the new string.
```

Comment:
Can we place the reference to the capitalized string object in some backticks? `String`.

Reply:
Do you think we can either include some info here about the difference between `CharSequence` and just `char`? I get that one is a "sequence of char values" but I noticed it's not mentioned in the [data types](codecademy.com/resources/docs/java/data-types)